homeassistant/components/ozw/cover.py

```diff
@@ -73,11 +74,22 @@ async def async_set_cover_position(self, **kwargs):
 
     async def async_open_cover(self, **kwargs):
         """Open the cover."""
-        self.values.primary.send_value(99)
+        self.values.open.send_value(PRESS_BUTTON)
+        self._open_button_pressed = True
 
     async def async_close_cover(self, **kwargs):
         """Close cover."""
-        self.values.primary.send_value(0)
+        self.values.close.send_value(PRESS_BUTTON)
+        self._close_button_pressed = True
+
+    async def async_stop_cover(self, **kwargs):
+        """Stop cover."""
+        if self._open_button_pressed:
+            self.values.open.send_value(RELEASE_BUTTON)
+            self._open_button_pressed = False
+        if self._close_button_pressed:
+            self.values.close.send_value(RELEASE_BUTTON)
+            self._close_button_pressed = False
```

Comment:
There seems to be a kind of idempotency on button press / release on openzwave / mqtt, which makes the "close" press not working if we only release the "open", hence the additional tracking of the pressed status of a button (a bit hacky). On the legacy `zwave/cover.py` implementation, using a release on the "open" button was sufficient.

What we can try to make the code simpler, is to send release on both buttons regardless of any tracking. It should make it work (and idempotency should not have any wrong effect then). I will also try scenario with local (physical button) push to see if a remote stop work.